scripts/menor.js

```diff
@@ -1,13 +1,17 @@
 
-function menor(){
-    var fullName = "Theophilus Ajiri Menor"; 
-    var hngId = "HNG-02809";
-    var langauge = "HTML, CSS";
-    var email = "ajirimenor@aim.com"
+var fullName = "Theophilus Ajiri Menor"; 
+var hngId = "HNG-02809";
+var langauge = "javascript";
+var email = "ajirimenor@gmail.com";
 
-    var myOutput = "Hello World, i am " + fullName + " with HNGi7 ID " + hngId + " and email " + email + " using " + langauge + " for stage 2 task";
+var myOutput = "Hello World, i am " + fullName + " with HNGi7 ID " + hngId + " and email " + email + " using " + langauge + " for stage 2 task";
+
+console.log(myOutput);
+function menorOutput(){
```

Comment:
There is no need for this function as you have used the myOutput variable in the previous line (line 9) to console.log your output.

Reply:
that is my old script, i have removed the function, waiting for you to merge the new one. I am sending another update now with script name menor_update.js

Reply:
This is the link of the one i corrected https://github.com/hng-teamfury/team-fury-1/blob/dev/scripts/menor.js